src/cascadia/LocalTests_SettingsModel/ColorSchemeTests.cpp

```diff
@@ -290,4 +291,81 @@ namespace SettingsModelLocalTests
             VERIFY_ARE_EQUAL(ARGB(0, 7, 7, 7), scheme2->_Background);
         }
     }
+
+    void ColorSchemeTests::UpdateSchemeReferences()
+    {
+        const std::string settingsString{ R"({
+                                                "defaultProfile": "Inherited reference",
+                                                "profiles": {
+                                                    "defaults": {
+                                                        "colorScheme": "Scheme 1"
+                                                    },
+                                                    "list": [
+                                                        {
+                                                            "name": "Explicit scheme reference",
+                                                            "colorScheme": "Scheme 1"
+                                                        },
+                                                        {
+                                                            "name": "Explicit reference; hidden",
+                                                            "colorScheme": "Scheme 1",
+                                                            "hidden": true
+                                                        },
+                                                        {
+                                                            "name": "Inherited reference"
+                                                        },
+                                                        {
+                                                            "name": "Different reference",
+                                                            "colorScheme": "Scheme 2"
+                                                        }
+                                                    ]
+                                                },
+                                                "schemes": [
+                                                    { "name": "Scheme 1" },
+                                                    { "name": "Scheme 2" },
+                                                    { "name": "Scheme 1 (renamed) " }
+                                                ]
+                                            })" };
+
+        auto settings{ winrt::make_self<CascadiaSettings>(false) };
+        settings->_ParseJsonString(settingsString, false);
+        settings->_ApplyDefaultsFromUserSettings();
+        settings->LayerJson(settings->_userSettings);
+        settings->_ValidateSettings();
+
+        // update all references from scheme 1 --> 3
+        const auto newName{ L"Scheme 1 (renamed) " };

```

Comment:
```suggestion
        const auto newName{ L"Scheme 1 (renamed)" };
```

having this space at the end seems _strange_. It also doesn't go from `1` to `3` as in the comment.